.github/workflows/build.yml

```diff
@@ -53,3 +65,57 @@ jobs:
             target/**/ornithe-installer-rs*
             !target/**/ornithe-installer-rs.d
             !target/**/build/
+  build_macos:
+    runs-on: ubuntu-latest
+    container: ghcr.io/rust-cross/cargo-zigbuild:0.20
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: stable
+      - uses: Swatinem/rust-cache@v2
+      - run: rustup target add aarch64-apple-darwin
+      - run: rustup target add x86_64-apple-darwin
+      - name: Build
+        run: cargo build --verbose --target universal2-apple-darwin
+      - name: Build release binary
+        run: cargo build --release --target universal2-apple-darwin
```

Comment:
Should be `cargo zigbuild ...` according to https://github.com/rust-cross/cargo-zigbuild